modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -378,12 +378,36 @@ definition {
 	}
 
 	@description = "Verify that the user can search for results inside Assigned to My Roles by Asset Type"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanSearchForAssetTypeOnAssignedToMyRolesTab {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
 
-		// TODO LRQA-68834 CanSearchForAssetTypeOnAssignedToMyRolesTab pending implementation
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry");
 
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Type(
+			locator1 = "MyWorkflowTasks#MY_WORKFLOW_TASKS_SEARCH_BAR",
+			value1 = "Blogs Entry");
+
+		Click(locator1 = "MyWorkflowTasks#MY_WORKFLOW_TASKS_SEARCH_BUTTON");
```

Comment:
You can take that piece and create a macro to make search at MyWorkflowTask area, see a example at [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/823/files).